Dockerfile_ams

```diff
@@ -54,7 +54,9 @@ RUN mkdir -p /opt/models/vehicle_detection_adas/1 && cd /opt/models/vehicle_dete
     wget https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-adas-0001/FP32/face-detection-adas-0001.bin && \
     wget https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-adas-0001/FP32/face-detection-adas-0001.xml
 COPY ie_serving /ie-serving-py/ie_serving
-COPY extras/ams_wrapper /ams_wrapper
+COPY extras/ams_wrapper/start_ams.py /ams_wrapper/start_ams.py
+COPY extras/ams_wrapper/src /ams_wrapper/src
+COPY extras/ams_wrapper/tests/invalid_startup /ams_wrapper/tests/invalid_startup
```

Comment:
why this test is included?

Reply:
It's actually not a test. This directory contains scripts that run AMS with invalid OVMS in the backend. It's used by the fixtures for functional tests.

Reply:
we need to add to to have this test run in functional tests.  am ok with it.